### ISSUE TITLE
python requests v. 1.2.x broken for some python versions

### ISSUE THREAD

**Issue by Profpatsch** [NixOS/nixpkgs#8895]
```python
>>> requests.get("https://google.de")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/nix/store/nq1lq5k4y77h2wfvfsm61f9myb7jz5xs-python3.4-requests-1.2.3/lib/python3.4/site-packages/requests/api.py", line 55, in get
    return request('get', url, **kwargs)
  File "/nix/store/nq1lq5k4y77h2wfvfsm61f9myb7jz5xs-python3.4-requests-1.2.3/lib/python3.4/site-packages/requests/api.py", line 44, in request
    return session.request(method=method, url=url, **kwargs)
  File "/nix/store/nq1lq5k4y77h2wfvfsm61f9myb7jz5xs-python3.4-requests-1.2.3/lib/python3.4/site-packages/requests/sessions.py", line 335, in request
    resp = self.send(prep, **send_kwargs)
  File "/nix/store/nq1lq5k4y77h2wfvfsm61f9myb7jz5xs-python3.4-requests-1.2.3/lib/python3.4/site-packages/requests/sessions.py", line 438, in send
    r = adapter.send(request, **kwargs)
  File "/nix/store/nq1lq5k4y77h2wfvfsm61f9myb7jz5xs-python3.4-requests-1.2.3/lib/python3.4/site-packages/requests/adapters.py", line 292, in send
    timeout=timeout
  File "/nix/store/nq1lq5k4y77h2wfvfsm61f9myb7jz5xs-python3.4-requests-1.2.3/lib/python3.4/site-packages/requests/packages/urllib3/connectionpool.py", line 423, in urlopen
    conn = self._get_conn(timeout=pool_timeout)
  File "/nix/store/nq1lq5k4y77h2wfvfsm61f9myb7jz5xs-python3.4-requests-1.2.3/lib/python3.4/site-packages/requests/packages/urllib3/connectionpool.py", line 238, in _get_conn
    return conn or self._new_conn()
  File "/nix/store/nq1lq5k4y77h2wfvfsm61f9myb7jz5xs-python3.4-requests-1.2.3/lib/python3.4/site-packages/requests/packages/urllib3/connectionpool.py", line 562, in _new_conn
    strict=self.strict)
TypeError: __init__() got an unexpected keyword argument 'strict'
>>> requests.__version__
'1.2.3'
```

Current version on pypi is 2.7.0.

Update: Oh, there is a `requests2`. Still, the v1 is broken.

Update 2: requests 1.2.1 patches python 3.3 compatability, so:
  - requests 1.2.1 – 1.2.3 should only be available in python 3.3
  - python34packages should only have requests2

**Comment by Profpatsch** [comment]
Should `python34Packages.requests2` be renamed to `python34Packages.requests` then?

**Comment by domenkozar** [comment]
fixed in c651e6cbe56ec5f9712044da3d6e41408056a3d9
